integrationTests/mock/peersHolderStub.go

```diff
@@ -0,0 +1,54 @@
+package mock
```

Comment:
can you move the stub into a p2p folder in testscommon and reuse from there wherever it is needed?
then you can remove the other duplicated implementations.

Reply:
done